Dockerfile.backend

```diff
@@ -1,10 +1,20 @@
-FROM node:18 as builder
 ARG ARCH=x86_64
 
-COPY . /src/
-RUN cd /src/ && \
-  yarn install && \
-  yarn build:backend
+FROM node:18 as pruner
```

Comment:
Is there a reason we don't use an alpine linux images?

Reply:
No, as discussed offline difftastic may be actually broken after such migration. I'll test it separately to this PR